docker/buildkite/docker-compose-opensearch2.yml

```diff
@@ -36,7 +36,7 @@ services:
           - elasticsearch
     environment:
       - discovery.type=single-node
-      - plugins.security.disabled=true
+      - "DISABLE_SECURITY_PLUGIN=true"
```

Comment:
did it just change env-var formats?  I lightly prefer the dot-separated one tbh.

Reply:
you can use dot-separated in `opensearch.yaml`, but for starting docker container with security disabled, you have to use this type of variable. Related documentation: https://github.com/opensearch-project/opensearch-build/blob/main/docker/release/README.md#disable-security-plugin-security-dashboards-plugin-security-demo-configurations-and-related-configurations